data-structures/binary_search_tree/test_bst.py

```diff
@@ -0,0 +1,45 @@
+# import pytest
+
+
+def test_tree_empty_instance(empty_bst):
+    """Check if root of binary tree is None"""
+    assert empty_bst.root is None
+
+
+def test_insert_empty(empty_bst):
+    """Check Binary tree root"""
+    empty_bst.insert('some random value')
+    assert empty_bst.root.val == 'some random value'
```

Comment:
This is the only test for `insert()` you have. You might want to also test that inserting into a filled tree puts the new value in the correct position.